packages/commonwealth/client/scripts/views/pages/Community/Groups/Create/CreateCommunityGroupPage.tsx

```diff
@@ -20,84 +26,97 @@ const CreateCommunityGroupPage = () => {
     return <PageNotFound />;
   }
 
-  return (
-    <GroupForm
-      formType="create"
-      onSubmit={(values) => {
-        const payload = {
-          chainId: app.activeChainId(),
-          address: app.user.activeAccount.address,
-          groupName: values.groupName,
-          groupDescription: values.groupDescription,
-          topicIds: values.topics.map((x) => x.value),
-          requirementsToFulfill:
-            values.requirementsToFulfill === 'ALL'
-              ? undefined
-              : values.requirementsToFulfill, // TODO: confirm if undefined means all requirements need to be satisfied
-          requirements: [],
-        };
+  const handleSubmit = (values: GroupResponseValuesType) => {
+    if (groups.length > 0) {
+      const foundGroup: Group = groups.find(
+        ({ name }) => name.toLowerCase() === values.groupName.toLowerCase()
+      );
+      if (foundGroup) {
+        notifyError('Group name already exists');
```

Comment:
I suggest we show the error next to the input like this

<img width="625" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/778b957d-dc7b-4c72-98dd-3ec9d53b7beb">

and show the error as we type instead of on submit as we do for other errors in the same form.

I think for this we can pass an "unallowedNames[]" array with existing group names to the `GroupForm` and for the `groupName` input we can listen to `onInput/onChange` and pass the `customError` if the name is duplicated (and also prevent onSubmit call). Lmk what you think would be best.

cc: @lzach83

Reply:
Addressed in my most recent commit